stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/HomeScreen.kt

```diff
@@ -84,3 +139,39 @@ fun HomeScreen(
         }
     }
 }
+
+private object NoRippleTheme : RippleTheme {
+    @Composable
+    override fun defaultColor() = Color.Unspecified
+
+    @Composable
+    override fun rippleAlpha(): RippleAlpha =
+        RippleAlpha(
+            0.0f, 0.0f,
+            0.0f, 0.0f
+        )
+}
+
+@Composable
+fun checkVisibility(viewModel: HomeViewModel): Boolean {
```

Comment:
this is presentation logic, it should be handled in the viewModel and return the status of the button